app/ui/browser-modern/views/browser/tabbar/tab.jsx

```diff
@@ -124,14 +125,16 @@ Tab.displayName = 'Tab';
 Tab.propTypes = {
   dispatch: PropTypes.func.isRequired,
   pageId: PropTypes.string.isRequired,
+  isPinned: PropTypes.bool.isRequired,
   isActive: PropTypes.bool.isRequired,
   isBeforeActive: PropTypes.bool.isRequired,
   isAfterActive: PropTypes.bool.isRequired,
 };
 
 function mapStateToProps(state, ownProps) {
   const page = PagesSelectors.getPageById(state, ownProps.pageId);
-  const pageIndex = PagesSelectors.getPageIndexById(state, ownProps.pageId);
+  const pageIndex = page ? PagesSelectors.getPageIndexById(state, ownProps.pageId) : -1;
```

Comment:
Correct, this should never happen. There are cases in other places where there might be  no selected tab momentarily, between when closing the last tab and replacing it with a new one. But not here.